resources/views/auth/application/questions.blade.php

```diff
@@ -66,8 +65,28 @@
                         <blockquote class="error">A státusz kitöltése kötelező</blockquote>
                         @enderror
                     </div>
-                    <div class="input-field col s12"><p style="margin-bottom:10px"><label style="font-size: 1em">Honnan
-                                hallott a Collegiumról?</label></p>
+                    <div class="input-field col s12">
+                        <p style="margin-bottom:10px">
+                                Megpályázni kívánt műhely(ek):
+                        </p>
+                        <div class="row">
+                        @foreach ($workshops as $workshop)
+                            <div class="col s6">
+                                @php $checked = $user->application->appliedWorkshops->contains($workshop->id) @endphp
+                                <x-input.checkbox only_input id="workshop_{{$workshop->id}}" :text="$workshop->name" name="workshop[]"
+                                                  value="{{ $workshop->id }}" checked='{{$checked}}'/>
+                            </div>
+                        @endforeach
+                        </div>
+                        @error('workshop')
+                        <blockquote class="error">@lang('user.workshop_must_be_filled')</blockquote>
+                        @enderror
+                        <blockquote>
+                        Kérjük, jelentkezését csak a szakmailag releváns műhelyekbe adja le. A műhelyek egymástól függetlenül dönthetnek a meghallgatásáról.
```

Comment:
I think this could imply that one can only apply into the workshop of their study lines at the university, which is not the case (as with comm&media). It could be rephrased like _"Kérjük, jelentkezését csak olyan műhelyekbe adja be, amelyek munkájában komolyan részt tud venni. A műhelyek egymástól függetlenül dönthetnek a meghallgatásáról."_

Reply:
"amelyek munkájában szakmailag részt tud venni"? Otherwise, agreed.

Reply:
Nice:)